README.md

```diff
@@ -68,37 +68,44 @@ The example app is a catalogue of the core components provided by this library.
+   Copy the `examples/discovery-search-app/.env` file to `examples/discovery-search-app/.env.local` file, and populate the following values from your Discovery project:
 
    ```
-   REACT_APP_PROJECT_ID=<project_id to query>
-   CLUSTER_USERNAME=<cluster username>
-   CLUSTER_PASSWORD=<cluster password>
-   CLUSTER_PORT=<cluster port>
-   CLUSTER_HOST=<cluster hostname>
+   REACT_APP_PROJECT_ID={REPLACE_ME}
+   CLUSTER_USERNAME={REPLACE_ME}
+   CLUSTER_PASSWORD={REPLACE_ME}
+   CLUSTER_PORT={REPLACE_ME}
+   CLUSTER_HOST={REPLACE_ME}
    ```
 
-   1. `REACT_APP_PROJECT_ID` is contained in the URL when viewing your Discovery project on the CP4D cluster (ex. `https://{CLUSTER_HOST}:{CLUSTER_PORT}/discovery/{RELEASE_NAME}/projects/{REACT_APP_PROJECT_ID}/workspace`)
-   2. `CLUSTER_USERNAME` the username used to log in to your CP4D dashboard and access your instance of Discovery
-   3. `CLUSTER_PASSWORD` the password used to log in to your CP4D dashboard and access your instance of Discovery
+   1. `REACT_APP_PROJECT_ID` is a guid contained in the URL (sample URL: `https://{CLUSTER_HOST}:{CLUSTER_PORT}/discovery/{RELEASE_NAME}/projects/{REACT_APP_PROJECT_ID}/workspace`) when viewing your Discovery project on the CP4D cluster (ex. `97ba736d-6563-4270-a489-c19d682b6369`)
+   2. `CLUSTER_USERNAME` the username used to log in to your CP4D dashboard and access your instance of Discovery (ex. `my_cp4d_username`)
+   3. `CLUSTER_PASSWORD` the password used to log in to your CP4D dashboard and access your instance of Discovery (ex. `my_cp4d_password`)
    4. `CLUSTER_PORT` defaults to `443` unless configured otherwise
-   5. `CLUSTER_HOST` the base URL of your CP4D cluster (ex. `{}.com`)
-
-For more information about configuring your Cloud Pak for Data cluster, see https://www.ibm.com/support/producthub/icpdata/docs/content/SSQNUZ_current/cpd/overview/overview.html
+   5. `CLUSTER_HOST` the base URL of your CP4D cluster (ex. `example.com`)
 
 5. Build the React components:
 
    ```
    yarn workspace @ibm-watson/discovery-react-components run build
    ```
 
-6. In one terminal, start the server:
+6. Perform one of the two steps
 
-   ```
-   yarn workspace discovery-search-app run server
-   ```
+   - Run the setup script (which does the same thing as the below step using the username/password provided in `.env.local`)
+     ```
+     yarn workspace discovery-search-app run server:setup
+     ```
+   - Create a `examples/discovery-search-app/.server.env` file with the following values:
+     ```
+     RELEASE_PATH={REPLACE_ME}
+     BASE_URL={REPLACE_ME}
+     ```
+     where:
+     - `RELEASE_PATH` is the url path part of the API URL shown in the CP4D UI (ex. `/discovery/release-name/instances/1578610482214/api`)
+     - `BASE_URL` is the protocol + host + port of the location that CP4D UI is hosted (ex. `https://zen-25-cpd-zen-25.apps.my-cluster-name.com:443`)
```

Comment:
This part seems confusing to me. Why would someone do the 2nd path?

Reply:
in case they don't want to run the server setup script which requires you to install `jq`

Reply:
Should we call that out? As a first-time user, I would not know why to choose one over the other. Or that I need to install `jq`.

Reply:
i think up above on line 44 we have the following prereq:
```
- jq (When running the server in the Discovery components example application)
  - `brew install jq`
```
^ i could either 
a) advocate for removing the dependency on jq and remove the first server:setup script entirely or
b) moving the dependency on jq to this section and remove this second option of manually crafting the server env file completely

Reply:
If `jq` is only needed for this setup script, then I think it best to move it to this section.